addon/components/flexberry-maplayer.js

```diff
@@ -340,6 +373,15 @@ let FlexberryMaplayerComponent = Ember.Component.extend(
     visibility: false,
 
     /**
+      Layer's opacity.
+
+      @property visibility
+      @type Int
+      @default 100
+    */
+    opacity: 100,
```

Comment:
Why in the model and in slider component opacity range is 0.01..1, but here is 0..10? It is strange.